src/router/index.js

```diff
@@ -219,11 +180,6 @@ let routes = [
     name: 'ContractDetailwithoutpage',
     component: ContractDetail
   },
-/*   {
-    path: '/contract/:contractType/:contractHash/:pageSize',
-    name: 'ContractDetailwithoutpageSize',
-    component: ContractDetail
-  }, */
```

Comment:
Was it already commented? It should work. Similar API is available.

Reply:
If so, I'll leave it commented.